pkg/vm/vm.go

```diff
@@ -1391,6 +1391,13 @@ func (v *VM) unloadContext(ctx *Context) {
 	if ctx.static != nil && currCtx != nil && ctx.static != currCtx.static {
 		ctx.static.Clear()
 	}
+	if ctx.CheckReturn {
```

Comment:
I don't think it should exist. We can do the same `estack` check in our code as C# node does.